docs/topics/eclipse-installing-plugin.adoc

```diff
@@ -44,4 +47,4 @@ endif::[]
 . Select all the *JBoss Tools - MTA* check boxes and click *Next*.
 . Review the installation details and click *Next*.
 . Accept the terms of the license agreement and click *Finish*.
-. Restart  Eclipse or CodeReady Studio.
+. Restart Eclipse or CodeReady Studio.
```

Comment:
I think that "Eclipse or CRS" could be replaced with "IDE" in each case. What do you think?

Reply:
I think it's better to leave your original phrasing alone. The document uses the product names more than it uses "IDE," so I'd suggest using the admittedly longer phrasing here.